e2e_tests/conftest.py

```diff
@@ -1,26 +1,11 @@
 import os
 
 import pytest
-from playwright.sync_api import Page, Playwright
+from playwright.sync_api import Page
 
 
-@pytest.fixture(params=["firefox", "chromium", "webkit"])
-def page(request, playwright: Playwright, browser_context_args) -> Page:
-    browser_type = request.param
```

Comment:
spoke with @colinbowen , creating our own browser instance here  meant that any cli flags we passed to the pytest playwright cmd werent getting picked up, e.g. specifying headed didnt make a difference. 

https://playwright.dev/python/docs/browsers shows that we can just specify the browsers we want to run on on with the browser flag, so I added that info to the readme. 

Most times locally we wont want to run all the tests against all browsers, we can save that for ci

Reply:
so do we have to pass parameter as browser every time we run the e2e test cases ?

Reply:
if you just want to run with the default chromium, no you dont need to pass a flag, only if you want multiple browsers or firefox or webkit

Reply:
ok